transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -129,17 +129,24 @@ extends ClientStateListener with JournalListener {
   }
   
   private def journalCommitEvent(entry: JournalEntry) {
-    if (state.streaming) {
+    if (state.streaming && entry.index >= state.index) {
+      if (entry.index != state.index) {
+        logger.warn(s"Commit skipped entries ${state.index} -> ${entry.index}")
```

Comment:
I mean the error message, it suggests that entries at `state.index` and `index` were skipped, while they actually were not, only the entries between them were